pages/queue.js

```diff
@@ -47,7 +47,8 @@ const Queue = (props) => (
 Queue.getInitialProps = async ({query},res) => {
     console.log(query,"::query")
     console.log(res,"::res")
-    const resp = await fetch('http://localhost:3000/queuedata')
+    const port = process.env.PORT || 5000;
+    const resp = await fetch(`http://localhost:${port}/queuedata`);
```

Comment:
Thanks for the PR! I think a better solution would be to take the hostname from the environment variable as well. Once I deploy it to wmflabs, I need to change it there.
So, make an environment variable and read the data from there.

Reply:
To get environment variables in the client, we will be needing webpack as well.

Reply:
It's okay. I am working on a patch for this. I will update you once it's done. It's more or less based on your solution. :)